src/SecureHeaders.php

```diff
@@ -957,16 +938,13 @@ private function importHeaders()
         $this->importStarted = true;
         # grab any headers that were already set and, if any,
         # add these to our internal header list
-        $this->headerBag = $this->httpAdapter->getHeaders();
+        $this->headers = $this->httpAdapter->getHeaders();
```

Comment:
This be `$headers`, as opposed to `$this->headers`? Perhaps we should name it `$newHeaders` or something for clarity too? 😜